### ISSUE TITLE
Provide Presentation & Tutorial as more specific types of EducationEvent

### ISSUE THREAD

**Issue by ptrelford** [schemaorg/schemaorg#1275]
An Event like an [Exhibition](http://schema.org/ExhibitionEvent), Conference or [Course](http://pending.1.schemaorgae.appspot.com/CourseInstance) may include sub events that make up the agenda via the [subEvent property](http://pending.1.schemaorgae.appspot.com/subEvent):

> An Event that is part of this event. For example, **a conference event includes many presentations**, each of which is a subEvent of the conference.

As per the notes on the subEvent property above, a conference, exhibition or course may contain presentations (or lectures) and tutorials (or practical classes).

To this ends I propose the following more specific types of EducationEvent:
- **Presentation**
- **Tutorial**

These new types may also provide the following additional optional properties:
- **topic** (Text)
- **track** (Text)

**Comment by RichardWallis** [comment]
In principle that sounds reasonable.

Naming would need to be looked at.  Presentation has several meanings including the awarding of something, and a PowerPoint (other brands are available) slide deck.  Also the meaning of tutorial seems to differ subtly in different geographies.

The new properties you reference also probably need similar attention _track_ feels like it should be some sort of _isPartOf_ relationship to a Course or CourseInstance, _topic_ could be replaced by _about_.

Also are these two the only ones, or are we going to end up proposing an ever increasing list of subtypes?

**Comment by ptrelford** [comment]
@RichardWallis thanks for the feedback.

I've suggested _Presentation_ as a sub type of EducationEvent as it is the nomenclature I've seen used at numerous conferences (along with _talks_ - but that seems even more ambiguous), and the nomenclature already used in description of the [subEvent property](http://pending.1.schemaorgae.appspot.com/subEvent). A less ambiguous name might be _Lecture_ in this case.

For defining a _track_ property for a presentation using the existing schema definition I've explored a number of options:
- superEvent with an event with a name matching the track name
- location using the place name as the track name
In both scenarios it works but is more verbose and less explicit than a specific property, but perhaps adequate.
